commitlint.config.js

```diff
@@ -13,9 +13,13 @@ module.exports = {
         'taboule',
         'docs',
         'guardoni',
-        'tktrex',
+        'tk:shared',
+        'tk:backend',
+        'tk:ext',
+        'yt:shared',
```

Comment:
sorry, but is it correct to have `tk:share`d/`yt:shared` when both of them should be `shared`?

Reply:
Yep, they're actually three different packages and I would keep them separated